packages/envd/internal/process/service.go

```diff
@@ -112,7 +112,7 @@ func (s *Service) scanRunCmdOut(pipe io.Reader, t output.OutType, process *Proce
 	}
 }
 
-func (s *Service) Start(id ID, cmd string, envVars *map[string]string, rootdir string) (ID, error) {
+func StartWithUser(s *Service, id ID, cmd string, envVars *map[string]string, rootdir string, user string) (ID, error) {
```

Comment:
I think by default all receiver methods are exposed as JsonRpc services, so I didn't want to expose two services.

If we have the startWithUser starting with lowercase it might be ok, but I haven't tested it.

This will also be different in the new rpc.